src/components/EventCard/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Card, ImageSide, EventImage, InfoSide, Title, Date, Description, UpperRow, LowerRow } from './style'
+import tempImg from '../../assets/imgs/logoandromedev.png'
+
+export const EventCard = ({
+  name = "Andromedev 2020",
+  init_date = "dd/mm/yyyy",
+  end_date = "dd/mm/yyyy",
+  org_image = tempImg,
+  description = "O Andromedev é um evento organizado pela organização estudantil OpenDevUFCG com o intuito de incentivar a participação de estudantes da Universidade Federal de Campina Grande (UFCG) em projetos open source. O evento ocorre em um período de nove semanas nas quais os estudantes colaboram com projetos open source através de um sistema de mentoria."
```

Comment:
Seria melhor colocar os props padrão no final do arquivo usando EventCard.defaultProps como está no arquivo do componente [Carousel](https://github.com/OpenDevUFCG/opendevufcg.org/blob/develop/src/components/Carousel/index.js)

```suggestion
export const EventCard = ({ name, init_date, end_date, org_image
```